src/util.h

```diff
@@ -104,4 +72,4 @@ namespace appimage {
             return (bool) ifs && ifs.good();
         }
     }
-}
```

Comment:
Good find, my IDE sometime refuses to notify me of missing trailing newline characters.